src/kolibri.yaml

```diff
@@ -0,0 +1,35 @@
+---
+description: >
+  The offline app for universal education.
+  Kolibri makes high quality education technology available in low-resource
+  communities such as rural schools, refugee camps, orphanages, non-formal
+  school systems, and prison systems.
+license: MIT
+name: Kolibri
+programmingLanguages:
+  - python
+  - javascript
+rating: 7
```

Comment:
I thought, we only rated 1-5?
If you want, we can do 1-10, though.

Reply:
:O That's a big confusion here. And we merged OpenFarm with a rating of 8 :p 
I feel that 1-5 is a bit too limited for a ranking, so let's go for 1-10 if you don't have an argument against :)